src/index.js

```diff
@@ -1,217 +1,41 @@
-import fsApi from 'fs';
-import path from 'path';
-import events from 'events';
+import scheduler from 'node-schedule';
 
-import config from 'config';
+import CGUs from './app/index.js';
+import * as logger from './logger/index.js';
+import Notifier from './notifier/index.js';
 
-import consoleStamp from 'console-stamp';
+(async () => {
+  try {
+    const serviceId = process.argv.slice(process.argv.indexOf('--serviceId'))[1];
+    const refilterOnly = process.argv.indexOf('--refilter-only') != -1;
+    const schedule = process.argv.indexOf('--schedule') != -1;
 
-import * as history from './history/index.js';
-import fetch from './fetcher/index.js';
-import filter from './filter/index.js';
-import loadServiceDeclarations from './loader/index.js';
+    const app = new CGUs();
+    await app.init();
+    await app.attach(logger);
```

Comment:
Why is attaching an event listener asynchronous? 🤔

Reply:
Move before init